src/core/sentry/index.ts

```diff
@@ -3,6 +3,15 @@ import { BrowserTracing } from '@sentry/tracing';
 
 import { version } from '../../../package.json';
 
+// Any error that we don't wanna send to sentry should be added here
+// via partial match
+const IGNORED_ERRORS = [
+  'Duplicate script ID',
```

Comment:
this one was fixed

Reply:
SInce it's #1 o #2 It won't hurt keep in it here in case it comes back